root/scripts/Audio.sh

```diff
@@ -1638,11 +1638,6 @@ ArtistTidalSearch () {
 	# $3 Lyric Type (true or false) - false = Clean, true = Explicit
 
 	# Get tidal artist album list
-	if [ ! -f /config/extended/cache/tidal/$2-videos.json ]; then
```

Comment:
Note that this block was removed because these files were not used anywhere else. Also I'm not sure if you intended to hardcode your x-tidal-token or not but I'm guessing not?

Reply:
Likely legacy, copy pasta from previous iterations of the script.